tools/dep_updaters/update-ada.sh

```diff
@@ -55,7 +55,7 @@ rm "$ADA_ZIP"
 curl -sL -o "$ADA_LICENSE" "https://raw.githubusercontent.com/ada-url/ada/HEAD/LICENSE-MIT"
 
 echo "Replacing existing ada (except GYP build files)"
```

Comment:
I think all the existing `Replacing existing * (except GYP build files)` comments in all the updater scripts need to be updated because now we are adding gn files to the list in addition to the existing gyp files.

Reply:
Updated.